.circleci/config.yml

```diff
@@ -23,19 +23,24 @@ jobs:
           name: Install dependencies
           command: |
             pip install --upgrade pip twine wheel
-            pip install --editable .[test] --upgrade
+            pip install --editable .[test,lint] --upgrade
           #  pip install --editable git+https://github.com/openfisca/openfisca-core.git@BRANCH_NAME#egg=OpenFisca-Core  # use a specific branch of OpenFisca-Core
 
       - save_cache:
           key: v1-py2-{{ checksum "setup.py" }}
           paths:
             - /tmp/venv/openfisca_france
 
+      - run:
+          name: Lint YAML files
```

Comment:
Same, why not put it in the `run test` part ?

Reply:
and why not run the linter on all yaml files (expecially the parameters ?)

Reply:
Also, it seems to me that a test with a unvalid yaml fails with openfisca-run-test. Is this to fail faster ?

Reply:
Un linter (comme flake8) fait plus que détecter les erreurs de syntaxe, il impose aussi des bonnes pratiques et une consistence qui permettent d'avoir un code plus propre.

Reply:
Les règles de gestion du `yamllint`me paraissent trop "relaxées" pour jouer ce rôle de bonne pratiques. C'est pour cela que je posait la question que peut être il était surtout utilisait comme validateur.

Reply:
@Anna-Livia je ne comprends pas [ta remarque](https://github.com/openfisca/openfisca-france/pull/1039/files#r202689397), as-tu un lien vers CircleCI ?

Reply:
Contrairement à ce qu'on peut passer `circleci tests glob "**/*.yaml"` correspond à `ls  **/*.yaml` et non pas `ls tests/**/*/yaml`. [L'outil fourni](https://circleci.com/docs/2.0/parallelism-faster-jobs/#globbing-test-files) par CircleCI est initialement fait pour les tests ; c'est de là que vient le préfix `circleci tests` dans les commandes.

Reply:
J'espérais que cette PR soit mergée dans la semaine et itérer sur une nouvelle base cette semaine. `yamllint` génére aujourd'hui beaucoup d'avertissements qui restent à être gérés. Une fois les modifications nécessaires apportées aux fichiers YAML les problèmes de format généreront des erreurs et non plus des avertissements.